### DISCUSSION TITLE
Problem with OpenID Connect Using Azure AD

### DISCUSSION THREAD

**Discussion by KernOil** [Budibase/budibase#6178] [Help] [answered]
After my last update to 1.0.194 from 1.0.101, I can not authenticate using a single sign-on. in AD I have everything correctly configured. if I paste my call-back URL in the URL bar I can get AD to authenticate and sign on. however, when selecting the single sign-on button from the frontend it takes me to a different page.  

URL returned from frontend: https://[DOMAIN]/api/global/auth/default/oidc/configs/[CONFIG STRING]
Working call-back URL https://[Domain]/api/global/auth/oidc/callback

any idea why they are different / how to fix the problem I am having?

**Comment by melohagan** [comment, marked as answer]
Hi @KernOil 

I believe this is a related bug raised here: https://github.com/Budibase/budibase/issues/5969

**Reply by KernOil** [reply]
Thank you !
